contract/test/test-contract.js

```diff
@@ -187,6 +189,14 @@ test('use the code that will go on chain to start the contract', async t => {
     });
 
     const { zoe } = t.context;
+    /**
+     * @param {{
+     *  installation: ERef<Installation<GameContractFn>>;
```

Comment:
`GameContractFn` seems overly specific here. But perhaps the relevant generic type is a pain.